assisted_service_mcp/src/utils/log_analyzer/signatures/base.py

```diff
@@ -75,12 +77,17 @@ def generate_table(data: Sequence[dict[str, Any]]) -> str:
         return tabulate(data, headers="keys", tablefmt="grid")
 
     @staticmethod
-    def format_time(time_str: str) -> str:
+    def format_time(time_str: str | datetime) -> str:
         """Format time string for display."""
-        try:
-            return dateutil.parser.isoparse(time_str).strftime("%Y-%m-%d %H:%M:%S")
-        except Exception:
-            return time_str
+        # Handle both datetime objects and ISO strings
```

Comment:
Why was this required? Is the type different depending on if we get the metadata from the API vs the log bundle?

Reply:
Yep it's not needed, thanks! Removed it.